src/data/features.ts

```diff
@@ -114,24 +114,24 @@ export const previewFiles = [
 			horizontal: 72,
 			vertical: 72
 		},
-		added: "Sunday, August 1, 2021 12:34 AM",
-		modified: "Monday, August 9, 2021 12:34 AM",
-		path: "C:\\Users\\Sara\\Pictures\\waves.png"
+		added: "Sunday, August 1, 2021 42:69 AM",
+		modified: "Monday, August 9, 2021 12:34 PM",
+		path: "C:\\Users\\Sara\\OneDrive\\Pictures\\waves.png"
```

Comment:
Can you revert the changes? The paths are fine as is.

Reply:
In pictures, the 'Sara' uses OneDrive, so I thought it would be appropriate to put that in as well.